cypress/models/index.ts

```diff
@@ -1,2 +1,3 @@
 export * from './dib-travel-accounts';
```

Comment:
Double-check A-Z order of exports

Reply:
I'm not sure I understand this one, it seems ok to me? I sorted the files I created..